py/desispec/scripts/proc.py

```diff
@@ -728,7 +730,7 @@ def main(args=None, comm=None):
                             os.makedirs(dirname)
                         desispec.scripts.specex.mean_psf(psfs,psfnightfile)
                 if os.path.isfile(psfnightfile) : # now use this one
-                    input_psf[camera] = psfnightfile
+                    input_psf[camera] = get_readonly_filepath(psfnightfile)
```

Comment:
Good point, though when I went to implement it I noticed that this whole section is wrapped in an `if False:` block left over from when desi_proc was trying to do multi-exposure tasks that eventually got replaced with desi_proc_joint_fit.  It's not strictly related to this PR but I'll remove that unused block here rather than craft read-only logic for code that is never run.